Use "service foo status" instead of "systemctl"

benjaoming opened the pull request:
Problem: `kolibri-server` freezes during installation on Devuan. I think this is the cause.

```
/etc/init.d/kolibri-server: 90: /etc/init.d/kolibri-server: systemctl: not found
INFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base
[ ok ] Starting nginx: nginx.
[uWSGI] getting INI configuration from /etc/kolibri/dist/uwsgi.ini
[ ok ] Reloading nginx configuration: nginx.
```

I tested this command and it works fine on Ubuntu and Devuan.

benjaoming committed:
Replace tabs with spaces and make indentation consistent

benjaoming committed:
Fix failure to run on non-systemd platforms

jredrejo reviewed (commented):
Why:
- are we going to change all these files to exchange tabs by spaces?
- are we going to discard systemd as a dependency? If there were good reasons to do it, what I could discuss, more changes should be needed them, in the way the package is installed. Any modern version of Debian or Ubuntu uses it and systemctl should always be available unless someone is using a version of Debian or Ubuntu we are not supporting.

benjaoming commented:
@jredrejo systemd was never a dependecy, either a system has sysvinit or systemd? Everything works fine, except the part that I already fixed.

Regarding tabs vs. spaces, the whole file was very hard to read with all the nested blocks, and yes isn't it nice that I also fixed that?

jredrejo commented:
> @jredrejo systemd was never a dependecy, either a system has sysvinit or systemd? Everything works fine, except the part that I already fixed.

If `sysvinit` is supported postinst needs to be modified adding  `update-rc.d` commands. 
If we are going to support both systems, `lsb-base` must be added as a dependency to make our life easier.
Since 2014 `systemd` is the default in Debian and Ubuntu. I have always considered systemd was needed but it doesn't need to be added to the dependencies because is an important part of the system. We can add it if we don't want to support two different systems with two different management cli's.

> 
> Regarding tabs vs. spaces, the whole file was very hard to read with all the nested blocks, and yes isn't it nice that I also fixed that?

That's a matter of taste. I don't like spending resources discussing that kind of things, but if you prefer it I think it should be in a different PR to avoid messing topics.

benjaoming commented:
I tested this on Devuan as I said.. so I don't know how to respond to the concerns about `update-rc.d` and `lsb-base`, yet.
